instrumentation/opentelemetry-instrumentation-aws-lambda/src/opentelemetry/instrumentation/aws_lambda/__init__.py

```diff
@@ -167,7 +168,7 @@ def _determine_parent_context(
     """
     parent_context = None
 
-    if not disable_aws_context_propagation:
+    if "_X_AMZN_TRACE_ID" in os.environ.get(OTEL_PROPAGATORS, ""):
```

Comment:
No, this should go away completely as the env var check will be done by the propagator instead.